---
title: "DNA Barcoding protocol"
description: |
  Protocol to be used to speciate trapped rodents from frozen samples. Protocol developed from DNA Barcoding in Mammals (Ivanova, Clare and Borisenko) from DNA Barcodes: Methods and Protocols.
author: "David Simons"
date: 05-20-2021
output:
  pdf_document:
    toc: true
    toc_depth: 2
#bibliography:
categories:
  - Lassa
---

## Rationale

"Express barcoding", a suitable method for ecological surveys and rapid taxonomic assessments will be used. The DNA extracts obtained from this method are not of archival quality and will generally be unidirectional short length (420 bp) which are sufficient for reliable species-level identification in mammals.

# Protocol
Adapted from [Ivanova et al.](https://www.researchgate.net/profile/Alex-Borisenko/publication/225287508_DNA_barcoding_in_mammals/links/02e7e5294b96c0e680000000/DNA-barcoding-in-mammals.pdf)

## Primer

BatL5310 + R6036R (need to check suitability) referencee is [here]((https://onlinelibrary.wiley.com/doi/10.1111/j.1471-8286.2007.01752.x))

## Materials

  + Tissue preservation
    1. FTA Elute blotting cards
  + Tissue subsampling and lysis
    1. Ethanol 96%
    2. Harris micropunch
    3. Microplate (Eppendorf twin-tec PCR plate, Fisher Scientific)
  + DNA extraction
    1. Swinging bucket centrifuge with microplate rotor
  + PCR
    1. 10% Trehalose: 5g of D-(+)-Trehalose dihydrate, molecular grade water to 50ml. Storage in 1-2ml aliquots at -20$^\circ$C
    2. 10x PCR Buffer, Minus Mg, -20$^\circ$C
    3. 50mM Magnesium Chloride, -20$^\circ$C
    4. 10mM Deoxynucleotide solution mix, store in 100$\mu$l aliquots at -20$^\circ$C
    5. 100$\mu$M primer stock: Dissolve desiccated primer in the amount of water indicated by the manufacturer to produce the final solution of 100$\mu$M
    6. 10$\mu$M primer stock (i.e. 20$\mu$l of 100$\mu$M stock diluted in 180$\mu$l of water)
    7. Platinum Taq DNA Polymerase, store at -20$^\circ$C
    8. Eppendorf Mastercycler ep gradient S Thermocycler
    9. Heat sealer
    10. Aluminium sealing gilm
    11. Clear sealing film
    12. Heat sealing film
    13. Hard-shell skirted microplate
    14. Swinging bucked centrifuge with microplate rotor
  + PCR product check
    1. Gel documentation system
    2. Pre-cast agarose gels
    3. E-base integrated power supply
  + Cycle sequencing reaction
    1. 10% Trehalose: 5g of D-(+)-Trehalose dihydrate, molecular grade water to 50ml. Storage in 1-2ml aliquots at -20C
    2. 5x Sequencing Buffer (400mM Tris-HCl pH 9 + 10 mM MgCl~2~).
    3. 100$\mu$M primer stock: Dissolve desiccated primer in the amount of water indicated by the manufacturer to produce the final solution of 100$\mu$M
    4. 10$\mu$M primer stock (i.e. 20$\mu$l of 100$\mu$M stock diluted in 180$\mu$l of water)
    5. BigDye Terminator v3.1 Cycle Sequencing Kit
    6. Microplate
    7. PCR Workstation
  + Sequencing reaction cleanup
    1. Sephadex G50
    2. Acroprep 96 filter plate, 0.45$\mu$ml GHP
    3. Pop-7 Polymer for 3730xl DNA Analyzers
    4. 3730xl DNA Analyzer Capillary Array, 50cm
    5. 10x Running Buffer for 3730xl DNA Analyzers
    6. Collection plate: MicroAmp 96-well reaction plate
    7. Hydroclave MC8 Steam Sterilizer
    8. PCR Workstation
    9. 3730xl DNA Anlyzer
    10. 8-Channel Matrix multichannel pipette
    
## Protocol

  + Tissue digestion [Boneparte et al.](https://journals.sagepub.com/doi/suppl/10.1177/0023677212473918/suppl_file/suppl-appendix.pdf)
    1. Proteinase K lysis in Sodium Dodecyl Sulphate containing buffer
      + Tissue biopsies are digested at 56$^\circ$C overnight in between 50 and 700$\mu$L of 10 mmol/L Tris-Cl pH 7.5, 100 mmol/L NaCl, 1mmol/L EDTA, 1% SDS, containing proteinase K (0.5mg/mL) and incubated for 15 min at 70$^\circ$C. Dilute between 1/20-1/50 in water as SDS disturbs the PCR and use 2$\mu$l for PCR reaction
    2. Place volume on FTA elute card.
    3. Use disposable tools to transfer the DNA from the samples to the card to prevent contamination
    4. Note the sample position on the card
    5. Leave card to air dry with the blotting portion opened
    6. Store the card away from direct light
  + Tissue lysis 
    1. Open FTA elute card and slide mat under blotting surface
    2. Punch FTA elute card using a 1.2mm micropunch
    3. Place one punch into each well of a 96 well microplatwe
    4. Clean micropunch between samples
  + DNA extraction
    1. Add 100$\mu$l of water to each well. Seal with aluminium film. Vortex for 10-15s and centrifuge for 1 min at 1,000*g*
    2. Aspirate and discard water from each well not disturbing the disks. Incubate open plate with disks at 56$^\circ$C for 5 min to evaporate residual water. After drying, disks can be used directly in PCR
  + PCR
    1. Select primer cocktails for express barcoding [BatL5310 + R6036R](https://onlinelibrary.wiley.com/doi/10.1111/j.1471-8286.2007.01752.x)
    2. Prepare PCR reagent mix, per well(12.5$\mu$l):
      + 6.25$\mu$l 10% Trehalose
      + 4$\mu$l ddH~2~O)
      + 1.25$\mu$l 10x buffer
      + 0.625 10mM MgCl~2~
      + 0.125$\mu$l 10$\mu$M primerA
      + 0.125$\mu$l 10$\mu$M primerB
      + 0.0625$\mu$l of 10mM dNTPs
      + 0.06$\mu$l of Platinum Taq (5U/$\mu$l)
    3. Keep Taq at -20$^\circ$C for as long as possible prior to use. Spin the tube with Taq by vortexing or pipetting.
    4. Dispence 12.5$\mu$l of PCR mix into each well of plate containing pre-washed FTA disks. Change pipette tips after each well.
  + PCR thermocycling
    1. Place aluminium foil or heat-seal cover over the top of 96-well plate and centrifuge for 1 minute at 1,000*g*
    2. Place the plate in a thermocycler block, close the lid and run the program "RatCOI"
    3. Apply thermocycling conditions as follows:
      + 94$^\circ$C for 2 min; 35 cycles (94$^\circ$C for 30s, 60$^\circ$C for 30s, and 72$^\circ$C for 1 min); final extension at 72$^\circ$C for 5 mins; then hold indefinitely at 10$^\circ$C
  + PCR product check
    1. Preset program ED on Mother E-base; with a run time of 4 min
    2. Remove gel from package and detach plastic comb, slide gel into electrode connections on Mother EBase
    3. Dispense 12$\mu$l of ddH~2~O into each E-ge; well
    4. Load 4$\mu$l PCR product into each E-gel well
    5. Commence electrophoresis. End of run is signalled by light flashing red after initially changing to green
    6. Remove gel from base and capture a digital image. 
    7. If success rate is >75%, all samples from the PCR plate should be sequenced, below 75% requires hit-picking of amplified products and failure tracking. 
  + Cycle sequencing
    1. Defrost the reagents. Big dye must not be exposed to light for more than a few minutes.
    2. Prepare "forward" and "reverse" sequencing mixes: combine forward primer with reagents in a single 2ml tube in the below proportions:
      + 104$\mu$l Primer
      + 195$\mu$l 2.5x SEQ Buffer
      + 26$\mu$l BigDye
      + 91$\mu$l ddH~2~O
      + 520$\mu$l 10% Trehalose
    3. Repeat for "reverse" primer
    4. Mix contents gently but thoroughly by pipetting
    5. Assemble "forward" and "reverse" sequencing plates:
      + Dispense 9$\mu$l of forwards-sequencing mix into each well of an empty 96 well plate - mark plate as forward
      + Repeat for reverse mix with different pipette tip - mark plate as reverse
    6. Add 1-1.5$\mu$l of non-purified PCR product from the PCR plate to each well of the plates using a new pipette tip each time
    7. Place aluminium foil or heat-seal cover over the plates and centrifuge at 1,000*g* for 1 min
    8. Place each sequencing plate in a thermocycler block and start program Seq3.1
      + Initial denaturation at 96$^\circ$C for 30s
      + Annealing at 55$^\circ$C for 15s
      + Extension at 60$^\circ$C for 4min
      + Hold at 4$^\circ$C
  + Sequencing cleanup
    1. Measure dry Sephadex G50 with black column loader into 350ml PALL filter plate. Prepare two plates to balance each other or prepare a single plate and appropriate counterbalance. 
    2. Add 300$\mu$l molecular grade water to plate wells prefilled with Sephadex powder; leave mixture to hydrate overnight at 4°C or for 3–4h at room temperature before use. 
    3. Join Sephadex plate with MicroAmp collection plate to catch water flow through and secure with two rubber bands.
    4. Balance two sets of plates for centrifuging; use additional rubber bands as needed.
    5. Centrifuge at 750*g* for 3 min to remove excess water and generate a packed Sephadex column
    6. Add the entire sequencing reaction to the center of Sephadex columns by pipette. Do not insert the pipette tip into the column; dispense liquid onto the upper surface of the column. Damaging the column with a pipette tip or dispensing solution onto the side of the column may result in incomplete removal of unincorporated BigDye terminators which will adversely affect sequencing results.
    7. Add 25$\mu$l of 0.1mM EDTA to each well of a new MicroAmp collection plate
    8. Elute clean sequencing reaction by attaching the collection plate containing EDTA to the bottom of Sephadex plate and securing with rubber bands.
    9. Balance two sets of plates for centrifuging; use additional rubber bands as needed.
    10. Centrifuge at 750*g* for 3min and remove Sephadex plate
    11. Cover the collection plate with septum
    12. Place the collection plate into black plate base of capillary sequencer and attach white plate retainer
    13. Stack assembled plates into ABI 3730xl capillary sequencer and import plate records using Plate Manager module of the Data Collection software
    14. Begin sequencing run
